src/main/java/net/greatstart/services/UserService.java

```diff
@@ -16,15 +18,19 @@
 @Transactional
 public class UserService {
 
+    private UserProfileMapper userMapper;
     private UserDao userDao;
     private RoleService roleService;
 
     @Autowired
-    public UserService(UserDao userDao, RoleService roleService) {
+    public UserService(UserProfileMapper userMapper, UserDao userDao, RoleService roleService) {
+        this.userMapper = userMapper;
         this.userDao = userDao;
         this.roleService = roleService;
     }
 
+    //ToDo: Rewrite method for create/delete User

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)